src/builtin.d

```diff
@@ -48,35 +49,35 @@ extern (C++) Expression eval_sin(Loc loc, FuncDeclaration fd, Expressions* argum
 {
     Expression arg0 = (*arguments)[0];
     assert(arg0.op == TOKfloat64);
-    return new RealExp(loc, sinl(arg0.toReal()), arg0.type);
+    return new RealExp(loc, TargetReal.sin(arg0.toReal()), arg0.type);
```

Comment:
Huh? We are talking about the interface required by the front-end (`core.builtin` etc.) here, which I prefer centralized in `real_t.d` and you'd prefer split between `real_t.d` (`real_t` type only) and `port.d`.
The new `port.d` is pretty short, and I'm not sure we'll need to adapt it for LDC (possibly for big-endian platforms).

I'm already working on a `real_t.d` implementation for LDC, based on @redstar's work around LLVM's `APFloat` type. That type doesn't seem to offer sin/cos/tan/sqrt, so that's going to be an issue - are these required built-ins available for the type you use in GDC?